### ISSUE TITLE
Allow React `propTypes` to be demonstrated

### ISSUE THREAD

**Issue by bebraw** [christianalfoni/webpack-bin#142]
For some reason the environment loses `propType` related warnings when using React. It would be valuable to have access to those. Otherwise it's not possible to demonstrate the idea and their value easily.

**Comment by christianalfoni** [comment]
Hm, I believe it runs production mode to reduce the size of the packages. Can remove that really

**Comment by bebraw** [comment]
Yeah, that would fix it. I thought it might be something like that.
